tests/gold_tests/connect/connect.test.py

```diff
@@ -106,3 +107,61 @@ def run(self):
 
 
 ConnectTest().run()
+
+
+class ConnectViaPVTest:
+    # This test also executes the CONNECT request but using proxy verifier to
+    # generate traffic
+    connectReplayFile = "replays/connect.replay.yaml"
+
+    def __init__(self):
+        self.setupOriginServer()
+        self.setupTS()
+
+    def setupOriginServer(self):
+        self.server = Test.MakeVerifierServerProcess(
+            "connect-verifier-server",
+            self.connectReplayFile)
+        # Verify server output
+        self.server.Streams.stdout += Testers.ExcludesExpression(
+            "uuid: 1",
+            "Verify the CONNECT request doesn't reach the server.")
+        self.server.Streams.stdout += Testers.ContainsExpression(
+            "GET /get HTTP/1.1\nuuid: 2", reflags=re.MULTILINE,
+            description="Verify the server gets the second request.")
+
+    def setupTS(self):
+        self.ts = Test.MakeATSProcess("connect-ts")
+
+        self.ts.Disk.records_config.update({
+            'proxy.config.diags.debug.enabled': 1,
+            'proxy.config.diags.debug.tags': 'http',
+            'proxy.config.http.server_ports': f"{self.ts.Variables.port}",
+            'proxy.config.http.connect_ports': f"{self.server.Variables.http_port}",
```

Comment:
I don't think these need to be strings. Can you try without making these string literals and see whether that works?

Reply:
Tried that but that didn't work out. I think they take string type to support specifying a range of ports.

Reply:
Gotcha. Thanks for trying it out.